apps/pl/scss/_scss2json.scss

```diff
@@ -29,13 +29,15 @@ $spacing-demo: $spacing;
 //@import '../_patterns/01-atoms/button/_button.scss';
 
 // Import the map of available custom font icons.
-@import '../../../source/_patterns/01-atoms/icon/scss/_icons-generated';
+@import '../../../source/_patterns/01-atoms/svgicon/scss/_icons-generated';
 
-$icons-demo: (
-  prefix: 'icon--',
-  fontname: 'iconfont',
-  icons: map-keys(map_get($__iconfont__data, 'iconfont')),
-);
+$svgicons: map-keys($sprites);
+
+// $icons-demo: (
```

Comment:
Do we want to nuke this?

Reply:
We need the svgicons variable to pass along to the JSON for the demo as well as the mapping function to create the CSS classes. We can delete the old icons-demo along with ripping out the old component.